hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/FrameRunnerTest.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
```

Comment:
The 4 tests in here take a long time to run on my laptop - 0.7-1.3s each.  That's _after_ JUnit startup time for the first test.  Wonder what takes so long in the JUnit environment?